go/api/glide_client.go

```diff
@@ -54,18 +57,18 @@ func (client *GlideClient) CustomCommand(args []string) (interface{}, error) {
 //
 // Return value:
 //
-//	"OK" if all configurations have been successfully set. Otherwise, raises an error.
+//	A StringValue containing "OK" if all configurations have been successfully set. Otherwise, raises an error.
 //
 // For example:
 //
 //	result, err := client.ConfigSet(map[string]string{"timeout": "1000", "maxmemory": "1GB"})
-//	result: "OK"
+//	result: api.StringValue{Val: "OK", IsNil: false}
```

Comment:
seems cumbersome to me.

I suggest that we add API:

```go
api.CreateStringValue("OK")
```

Reply:
Done.